packages/castable-video/castable-mixin.js

```diff
@@ -138,6 +140,14 @@ export const CastableMediaMixin = (superclass) =>
       mediaInfo.metadata.title = this.title;
       mediaInfo.metadata.images = [{ url: this.poster }];
 
+      if (isHls(this.castSrc)) {
+        const isM4SSegment = (await getPlaylistSegmentFormat(this.castSrc)) === 'm4s';
```

Comment:
just wondering if `m4s` is always the value for all mp4 variants? mp4, m4a

Reply:
I'll review the spec to check if there are other possible extensions that we might receive. This is part of the tests I still need to complete